src/components/report-filter-controls.tsx

```diff
@@ -55,149 +55,172 @@ function ReportFilterControls({ controls }: ReportFilterControlsProps): VNode {
 
   return (
     <>
-      <form ref={formRef} onChange={update} className="usa-form">
-        <div>
-          <label>
-            Start
-            <input
-              type="date"
-              name="start"
-              value={yearMonthDayFormat(start)}
-              className="usa-input"
-            />
-          </label>
-        </div>
-        <div>
-          <label>
-            Finish
-            <input
-              type="date"
-              name="finish"
-              value={yearMonthDayFormat(finish)}
-              className="usa-input"
-            />
-          </label>
-        </div>
-        <div>
-          <button type="button" className="usa-button" onClick={updateTimeRange(utcWeek, -1)}>
-            &larr; Previous Week
-          </button>
-          <button type="button" className="usa-button" onClick={updateTimeRange(utcWeek, +1)}>
-            Next Week &rarr;
-          </button>
-        </div>
-        {controls?.includes(Control.IAL) && (
-          <fieldset className="usa-fieldset">
-            <legend className="usa-legend">IAL</legend>
-            <div className="usa-radio">
-              <input
-                type="radio"
-                id="ial-1"
-                name="ial"
-                value="1"
-                checked={ial === 1}
-                className="usa-radio__input"
-              />
-              <label htmlFor="ial-1" className="usa-radio__label">
-                IAL 1
-              </label>
-            </div>
-            <div className="usa-radio">
-              <input
-                type="radio"
-                id="ial-2"
-                name="ial"
-                value="2"
-                checked={ial === 2}
-                className="usa-radio__input"
-              />
-              <label htmlFor="ial-2" className="usa-radio__label">
-                IAL2
-              </label>
+      <form ref={formRef} onChange={update} className="usa-form-full-width">
+        <div className="grid-container padding-0">
+          <div className="grid-row grid-gap">
+            <div className="tablet:grid-col-6">
+              <fieldset className="usa-fieldset">
+                <legend className="usa-legend">Time Range</legend>
+                <div>
+                  <label>
```

Comment:
I was thinking these labels could use a bit of margin. It looks like we'd get some if we used the design system class:

```suggestion
                  <label class="usa-label">
```

Reply:
love it! added to all of them in 99c46aca3d008da2a0a3f6a26ac65dd5526866d8